package.json

```diff
@@ -55,7 +55,7 @@
     "babel-register": "^6.5.1",
     "chai": "^3.5.0",
     "enzyme": "^1.5.0",
-    "jsdom": "^6.5.1",
+    "jsdom": "^8.0.3",
```

Comment:
we may want to do `^6.5.1 || ^7.x.y || ^8.0.3` if in fact they all work - that would silence greenkeeper til v9 comes out